docs/source/architectural_design.rst

```diff
@@ -0,0 +1,54 @@
+Architectural design
+====================
+
+Overview
+--------
+
+Given a configuration file (Data Resource Schema) VIA a REST API the Data Resource Generator (DRG) will dynamically generate:
+
+#. Database tables with relationships
+    * SQLAlchemy Object Relation Mappings (ORM)
+    * Generate and run database migrations
+#. A RESTful API
+    * Running Flask application
+    * Open API Specification 3.0
+
+The application is built with Python and uses Flask.
```

Comment:
I suggest removing passive constructions when you can. This seems like a good section to use "we" and "BrightHive." I suggest condensing lines 16-22 into a clear statement of dependencies, something like:

> BrightHive built the DRG with Python and several other libraries. These include:
>
> **Table Schema and Swagger.** These open data specifications provide the backbone of a Data Resource Schema, which embeds table schema, descriptors, a swagger specification, and metadata. 
> **Flask.** The DRG application itself is a Flask API. The API has a generation route that holds a reference to the Flask object.